src/api/archive_unstable_body.md

```diff
@@ -2,7 +2,7 @@
 
 **Parameters**:
 
-- `hash`: String containing the hexadecimal-encoded hash of the header of the block whose body must be retrieved.
+- `hash`: String containing an hexadecimal-encoded hash of the header of the block whose body to fetch.
```

Comment:
Nit: Ideally I think it would read 

```suggestion
- `hash`: String containing a hexadecimal-encoded hash of the header of the block whose body will be retrieved.
```

But whatever it is, it's good to be consistent!